test/test_grid.py

```diff
@@ -935,3 +935,12 @@ def test_antimeridian_distance_nodes(self):
     def test_antimeridian_distance_face_centers(self):
         """TODO: Write addition tests once construction and representation of face centers is implemented."""
         pass
+
+
+class TestGridFromVertices(TestCase):
+
+    def test_spherical_voronoi(self):
+        verts = [(0, 0), (45, 45), (-30, 150), (60, -30), (-75, -120), (90, 0),
+                 (-90, 90), (30, -60)]
+        grid_from_verts = ux.open_grid(verts)
+        grid_from_verts.from_vertices(method="spherical_voronoi")
```

Comment:
Two scenarios:

1. ux.open_grid(verts, build_connectivity="triangular" or "voronoi") - this automatically returns a grid with elements
2. grid_with_verts_only = ux.open_grid(verts)  -> 

make the 2nd 
`call grid_with_verts_conn = grid_with_verts_only.from_vertices(method="spherical_voronoi")`
we can use the same object as done above? which one is better?

Reply:
I think the first one might make the most sense. If we detect only points being sent in, would it make sense to create the connectivity by default?

Reply:
> I think the first one might make the most sense. If we detect only points being sent in, would it make sense to create the connectivity by default?
+1